test/ecto/repo_test.exs

```diff
@@ -230,7 +236,7 @@ defmodule Ecto.RepoTest do
   end
 
   test "insert!, update!, insert_or_update! and delete! fail on changeset with wrong action" do
-    invalid = %Ecto.Changeset{valid?: true, data: %MySchema{}, action: :other}
+    invalid = %Ecto.Changeset{valid?: true, data: %MySchema{id: 123}, action: :other}
```

Comment:
We can skip this change by moving line with `filters` down to line 245, but I don't think that we need to test on stale changeset here.